ios/Classes/Blemulator/Constants/SimulationChannelName.m

```diff
@@ -0,0 +1,4 @@
+#define BASE @"com.polidea.blemulator"
+
+NSString * const SIMULATION_CHANNEL_NAME_TO_JAVA = (BASE @"/toJava");
```

Comment:
is there any channel to JAVA?

Reply:
It's a Dart channel name that was previously introduced when development was ongoing for Android version, so I'm just sticking to the convention here.

 It was discussed that in the future it might need to be changed to "toPlatform" to be more appropriate for Multiplatform solution (might need to create a separate issue for that task).

Reply:
as I understand the value cannot be changed but I suppose the name(`SIMULATION_CHANNEL_NAME_TO_JAVA`) could be, am I right?

Reply:
Good point, changed to `SIMULATION_CHANNEL_TO_PLATFORM`.